madara/crates/client/db/src/block_db.rs

```diff
@@ -424,4 +424,24 @@ impl MadaraBackend {
             }
         }
     }
+
+    pub fn get_starting_block(&self) -> Option<u64> {
+        self.starting_block
+    }
+
+    pub async fn get_sync_status(&self) -> SyncStatus {
+        self.sync_status.read().await.clone()
+    }
+
+    pub async fn set_sync_status(&self, highest_block_n: Option<u64>, highest_block_hash: Option<Felt>) {
```

Comment:
why not sync_status: SyncStatus directly instead of 2 options?

this would be less confusing since it's unclear what passing None, Some, Some None or None, None would mean here